tests/foreman/cli/test_remoteexecution.py

```diff
@@ -1019,3 +1020,204 @@ def test_positive_rex_against_infra_hosts(
         )
         result = JobInvocation.info({'id': invocation_command['id']})
         assert result['success'] == '1'
+
+
+class TestPullProviderRex:
+    """Tests related to remote execution via pull provider (mqtt)"""
+
+    @pytest.mark.tier3
+    @pytest.mark.upgrade
+    @pytest.mark.rhel_ver_match('[^6].*')
+    def test_positive_run_job_on_host_converted_to_pull_provider(
+        self,
+        module_org,
+        smart_proxy_location,
+        module_ak_with_cv,
+        module_capsule_configured_mqtt,
+        rhel_contenthost,
+    ):
+        """Run custom template on host converted to mqtt
+
+        :id: 9ad68172-de7b-4578-a3ae-49b6ff461691
+
+        :expectedresults: Verify the job was successfully ran against the host converted to mqtt
+
+        :CaseImportance: Critical
+
+        :parametrized: yes
+        """
+        result = rhel_contenthost.execute(
+            f'curl -o /etc/pki/ca-trust/source/anchors/satellite-sat-engineering-ca.crt \
+                    {settings.repos["DOGFOOD_REPO_HOST"]}/pub/katello-server-ca.crt; \
+                    update-ca-trust'
+        )
+        assert result.status == 0, 'Failed to download certificate'
```

Comment:
Using `command; command`, if `curl` failed but `update-ca-trust` was successful, this pass will check.